packages/ui/src/core/check-box-group/types/check-box-group.mixin.ts

```diff
@@ -0,0 +1,18 @@
+import { WithTheme } from '@scaleflex/ui/theme/entity';
+import { FontVariant as FV } from '@scaleflex/ui/utils/types/typography';
+
+import { Size } from '../../check-box/size';
+
+export const sizeCheckboxGroupMixin = {
```

Comment:
Why put this in checkbox group. 
A note, any group components are wrapper to the component itself. and we add extra stuff in it like adding label, hint. For example
InputGroup are a wrapper to input itself

So all core logic for a component shouldn't be in a group component